test/metadata-routes.test.ts

```diff
@@ -261,58 +238,36 @@ describe('API Endpoints for viewing dataset objects', () => {
-                }
-                fileImport.location = DataLocation.DataLake;
-                await fileImport.save();
-
-                const res = await request(app)
-                    .get(`/dataset/${dataset1Id}/revision/by-id/${revision1Id}/import/by-id/${import1Id}/raw`)
-                    .set(getAuthHeader(user));
-                expect(res.status).toBe(500);
-                expect(res.body).toEqual({
-                    success: false,
-                    status: 500,
-                    errors: [
-                        {
-                            field: 'csv',
-                            message: [
-                                {
-                                    lang: Locale.English,
-                                    message: t('errors.download_from_datalake', { lng: Locale.English })
-                                },
-                                {
-                                    lang: Locale.Welsh,
-                                    message: t('errors.download_from_datalake', { lng: Locale.Welsh })
-                                }
-                            ],
-                            tag: { name: 'errors.download_from_datalake', params: {} }
-                        }
-                    ],
-                    dataset_id: dataset1Id
-                });
-            });
+            // test('Get file from a revision and import returns 500 if an error with the Data Lake occurs', async () => {
+            //     DataLakeService.prototype.getFileStream = jest
+            //         .fn()
+            //         .mockRejectedValue(new Error('Unknown Data Lake Error'));
+            //     const res = await request(app)
+            //         .get(`/dataset/${dataset1Id}/revision/by-id/${revision1Id}/import/by-id/${import1Id}/raw`)
+            //         .set(getAuthHeader(user));
+            //     expect(res.status).toBe(500);
+            //     expect(res.body).toEqual({
+            //         success: false,
+            //         status: 500,
+            //         errors: [
+            //             {
+            //                 field: 'csv',
+            //                 message: [
+            //                     {
+            //                         lang: Locale.English,
+            //                         message: t('errors.download_from_datalake', { lng: Locale.English })
+            //                     },
+            //                     {
+            //                         lang: Locale.Welsh,
+            //                         message: t('errors.download_from_datalake', { lng: Locale.Welsh })
+            //                     }
+            //                 ],
+            //                 tag: { name: 'errors.download_from_datalake', params: {} }
+            //             }
+            //         ],
+            //         dataset_id: dataset1Id
+            //     });
+            // });
```

Comment:
Is this being left in for later or can it be removed completely?

Reply:
Its being left for now... The test seems to hang and I can't quite figure out why... But this end point is only used by us for development purposes so I feel leaving it temporarily and revisiting when there's a bit more time makes sense.

Reply:
Resolved... Test fixed.  The issue occurs when a route just returns a status and no body... This can cause the test to hang as the response hasn't resolved properly.